test/e2e/utils/utils.go

```diff
@@ -259,6 +259,32 @@ func TestWebhookServerTLS(ctx context.Context, f *framework.Framework, serviceNa
 	return nil
 }
 
+func TestPluginWithOperator(deviceName string, resourceNames []v1.ResourceName, f *framework.Framework, ns string, ctx context.Context, timeout time.Duration) {
```

Comment:
this can be local to `test/e2e/operator`

Reply:
done!